crates/stackable-certs/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [0.4.0] - 2024-07-09
```

Comment:
note: All the release dates need to be updated to today.

Reply:
1c4e7b5